Dockerfile_mbtci_template

```diff
@@ -270,7 +271,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && curl -fsSLO --compressed "https://github.com/CycloneDX/cyclonedx-cli/releases/download/v${CYCLONEDX_CLI_VERSION}/${CYCLONEDX_CLI_BINARY}-linux-${ARCH}" \
   && chmod a+rx ${CYCLONEDX_CLI_BINARY}-linux-${ARCH} \
   && mv ${CYCLONEDX_CLI_BINARY}-linux-${ARCH} /usr/local/bin/${CYCLONEDX_CLI_BINARY} \
-  && apt-get remove --purge --autoremove -y ca-certificates curl gnupg dirmngr \
+  && apt-get remove --purge --autoremove -y gnupg dirmngr \
```

Comment:
Security policy enforce that only toolchains must be part of the available binaries. Binaries allowing to ease HTTP, FTP, ... request from the build env must be removed.

Reply:
Thank you for the hint.